src/keri/vdr/eventing.py

```diff
@@ -1644,14 +1644,15 @@ def processQuery(self, serder, source=None, sigers=None, cigars=None):
                 self.cues.append(dict(kin="replay", src=src, dest=source.qb64, msgs=msgs))
         elif route == "tsn":
             ri = qry["ri"]
+            src = qry["src"]
             if ri in self.tevers:
                 tever = self.tevers[ri]
                 tsn = tever.state()
-                self.cues.push(dict(kin="reply", route="/tsn/registry", data=asdict(tsn), dest=source))
+                self.cues.push(dict(kin="reply", route=f"/tsn/registry/{src}", data=asdict(tsn), dest=source.qb64))
```

Comment:
Curious if we should just skip this if `vci` is set below, depends on if we need to escrow or not